### DISCUSSION TITLE
Cannot install @visx/xychart

### DISCUSSION THREAD

**Discussion by tonyneel923** [airbnb/visx#1449] [Q&A]
ERESOLVE unable to resolve dependency tree. Looks like mismatching react version requirements in the dependencies. Anyone else have this issue?

**Comment by williaster** [comment]
Hmm, the codesandbox seems to be working without error.  Maybe its versions will help in debugging?

https://codesandbox.io/s/github/airbnb/visx/tree/master/packages/visx-demo/src/sandboxes/visx-xychart
